CONTRIBUTING.md

```diff
@@ -28,8 +31,10 @@ Running integration tests locally requires:
   see the related Azure [documentation](https://learn.microsoft.com/en-us/java/api/overview/azure/identity-readme?view=azure-java-stable#defaultazurecredential).
 
 * A running postgres:
-```
- ./library/local-dev/run_postgres.sh start|stop
+```sh
+./scripts/setup
```

Comment:
see comment in `service/README.md` below - happy to consolidate conversation in that thread (commenting to bookmark instance)

Reply:
... based on comments above, resolving this part of the doc.